frontend/src/components/forms/CreateLog.tsx

```diff
@@ -0,0 +1,394 @@
+                                            onChange={handleBuildingChange}
+                                            styles={{
+                                                control: (provided, state) => ({
+                                                    ...provided,
+                                                    border: getBorderStyle(state, buildingError),
+                                                    "&:hover": {
+                                                        borderColor: buildingError ? "#e53e3e" : "#B1B1B1",
+                                                        cursor: "pointer"
+                                                    },
+                                                    borderRadius: "4px"
+                                                })
+                                            }}
+                                        />
+                                        <FormErrorMessage>Building is required.</FormErrorMessage>
+                                    </FormControl>
+                                </Col>
+                                <Col>
+                                    <FormControl isRequired isInvalid={residentError} mt={4}>
+                                        <FormLabel>Resident</FormLabel>
+                                        <Select
+                                            options={RESIDENTS}
+                                            placeholder="Select Resident"
+                                            onChange={handleResidentChange}
+                                            styles={{
+                                                control: (provided, state) => ({
+                                                    ...provided,
+                                                    border: getBorderStyle(state, residentError),
+                                                    "&:hover": {
+                                                        borderColor: residentError ? "#e53e3e" : "#B1B1B1",
+                                                        cursor: "pointer"
+                                                    },
+                                                    borderRadius: "4px"
+                                                })
+                                            }}
+                                        />
+                                        <FormErrorMessage>Resident is required.</FormErrorMessage>
+                                    </FormControl>
+                                </Col>
+                            </Row>
+
+                            <Row>
+                                <Col>
+                                    <FormControl mt={4}>
+                                        <FormLabel>Tags</FormLabel>
+                                        <Select
+                                            options={TAGS}
```

Comment:
If you have too much time on your hands, could put this in state and set it with the test data in a useEffect when the component renders